CHANGELOG.md

```diff
@@ -3,6 +3,27 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
+2.7.0
+=====
```

Comment:
we need to use ----- instead of ===== otherwise it uses a different style for the header